Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```
@@ -33,10 +33,16 @@ spec:
     coreLimit: "1000m"
     memory: "512m"
     labels:
-      version: 3.5.0
+      version: 3.5.1
+    volumeMounts:
+      - name: shared-volume
+        mountPath: /mounts/shared-volume/shared
   executor:
     cores: 1
     instances: 2
     memory: "512m"
     labels:
-      version: 3.5.0
+      version: 3.5.1
+    volumeMounts:
+      - name: shared-volume
+        mountPath: /mounts/shared-volume/shared
```

nit: Let's also add the volume specification, Airflow can add it if not specified, but let's add them also explicitly. Because usually `volumeMounts` and `volumes` are specified together. The example:

```suggestion
    volumeMounts:
      - name: shared-volume
        mountPath: /mounts/shared-volume/shared
    volumes:
      - name: shared-volume
        persistentVolumeClaim:
          claimName: kubeflow-shared-pvc
```

We can do it for all yaml files here